flake.nix

```diff
@@ -240,5 +248,23 @@
             };
           };
       }
-    );
+    ) //
+    {
+      nixosModule = { config, ... }:
+        {
+          config = {
+            boot.kernelPatches = [{
+            name = "memflow-enable-kallsyms";
+            patch = null;
+            extraConfig = ''
+                KALLSYMS y
+                KALLSYMS_ALL y
+              '';
+            }];
+
+            boot.kernelModules = [ "memflow" ];
```

Comment:
We should leave it up to the user if they want to have the module load on boot or not.
```suggestion
```